Dockerfile

```diff
@@ -25,10 +25,6 @@ RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
 COPY --from=builder /app/dist /action-release/dist/
 RUN chmod +x /action-release/dist/index.js
 
-# move the sentry-cli binary to where the entrypoint expects it
-RUN mv /action-release/node_modules/@sentry/cli/sentry-cli /action-release/sentry-cli
```

Comment:
@armenzg @asottile-sentry Now that I've updated the CLI, I can remove this ugly hack 😄